### PR TITLE
Update Windows Defender Removal Documentation

### PR DESCRIPTION
**Pull request by emtuls**
This updates the documentation for removing/disabling Windows Defender, fixing the issue described in this PR here: https://github.com/mandiant/flare-vm/pull/530

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,10 @@ This section documents the steps to install FLARE-VM. You may also find usuful t
     * Disable Windows Updates (at least until installation is finished)
       * https://www.windowscentral.com/how-stop-updates-installing-automatically-windows-10
     * Disable Tamper Protection and any Anti-Malware solution (e.g., Windows Defender), preferably via Group Policy.
-      * [https://stackoverflow.com/questions/62174426/how-to-permanently-disable-windows-defender-real-time-protection-with-gpo](https://superuser.com/a/1757341)
+      * GPO: [https://stackoverflow.com/questions/62174426/how-to-permanently-disable-windows-defender-real-time-protection-with-gpo](https://superuser.com/a/1757341)
+      * Non-GPO - Manual: [https://www.maketecheasier.com/permanently-disable-windows-defender-windows-10/](https://www.maketecheasier.com/permanently-disable-windows-defender-windows-10/)
+      * Non-GPO - Automated: [https://github.com/ionuttbara/windows-defender-remover](https://github.com/ionuttbara/windows-defender-remover)
+      * Non-GPO - Semi-Automated (User needs to toggle off Tamper Protection): [https://github.com/AveYo/LeanAndMean/blob/main/ToggleDefender.ps1] (https://github.com/AveYo/LeanAndMean/blob/main/ToggleDefender.ps1)
```
Comment (Ana06) [inline README.md:33]:
```suggestion
      * Non-GPO - Manual: https://www.maketecheasier.com/permanently-disable-windows-defender-windows-10
      * Non-GPO - Automated: https://github.com/ionuttbara/windows-defender-remover
      * Non-GPO - Semi-Automated (User needs to toggle off Tamper Protection): https://github.com/AveYo/LeanAndMean/blob/main/ToggleDefender.ps1
```

In GH markdow the `[]()` syntax is not needed for an URL, so I think we should remove it for all that are a duplication (text and link are the same). In the first link, there are two different URLs, the one with the StackOverflow question and the link to the answer. I am ok with either leaving both (as it is) or change it by https://superuser.com/a/1757341. I haven't included it in the code suggestion.

**Review by Ana06 — COMMENTED** [review]
Thanks for coming up with this proposal @emtuls! LGTM apart from the nitpick with the URLs format.

@0xdf223 is ok for you that we close https://github.com/mandiant/flare-vm/pull/530 in favor of this PR?

**Comment by 0xdf223** [comment]
Works for me! Thanks for the followup.

File: README.md [hunk 2]
```diff
@@ -27,7 +27,10 @@ This section documents the steps to install FLARE-VM. You may also find usuful t
     * Disable Windows Updates (at least until installation is finished)
       * https://www.windowscentral.com/how-stop-updates-installing-automatically-windows-10
     * Disable Tamper Protection and any Anti-Malware solution (e.g., Windows Defender), preferably via Group Policy.
-      * [https://stackoverflow.com/questions/62174426/how-to-permanently-disable-windows-defender-real-time-protection-with-gpo](https://superuser.com/a/1757341)
+      * GPO: https://superuser.com/a/1757341
+      * Non-GPO - Manual: https://www.maketecheasier.com/permanently-disable-windows-defender-windows-10/
```
Comment (Ana06) [inline README.md:31]: [nitpick] 
```suggestion
      * Non-GPO - Manual: https://www.maketecheasier.com/permanently-disable-windows-defender-windows-10
```

**Review by Ana06 — APPROVED** [review]
Thanks @emtuls! :smile:

**Pull request merged**